exporter/opentelemetry-exporter-opencensus/pyproject.toml

```diff
@@ -28,7 +28,7 @@ classifiers = [
 dependencies = [
   "grpcio >= 1.0.0, < 2.0.0",
   "opencensus-proto >= 0.1.0, < 1.0.0",
-  "opentelemetry-api >= 1.17.0.dev",
+  "opentelemetry-api >= 1.18.0.dev",
```

Comment:
i think this shouldn't be updated updated to `1.17.0` or something like that, since it is not strictly pinned (`==`) to the dev version.

Reply:
The script is broken. I plan to address this as part of https://github.com/open-telemetry/opentelemetry-python/issues/3183.

Reply:
This is tricky; if we update this to >= `1.17.0`, the CI will not find the version because it has not yet been released. But to release, both pull requests should be merged. I will bring this up in the next SIG meeting, and we will find a better way to manage this.